python/cudf/cudf/utils/_numba.py

```diff
@@ -7,6 +7,19 @@
 
 from numba import config as numba_config
 
+try:
+    from pynvjitlink.patch import (
+        patch_numba_linker as patch_numba_linker_cuda_12,
+    )
+except ModuleNotFoundError:
```

Comment:
I think you can use the more generic `ImportError` (parent class of `ModuleNotFoundError`).
```suggestion
except ImportError:
```